### ISSUE TITLE
Compass ignores the generated_images_dir

### ISSUE THREAD

**Issue by dirkschmid** [Compass/compass#1162]
I have an issue where I'm setting a generated_images_dir in my config.rb that specifies another folder for generated images than the default images folder. the config is a follows:

``` ruby
images_dir = "images"
sass_dir = "styles"

relative_assets = true

css_dir = "stylesgen"
generated_images_dir = "imagesgen"
```

my sprite-map is defined like this:

``` ruby
$icons: sprite-map("../images/foo/bar/icons/*.png")
```

So what compass now does is that it generates the folder "imagesgen" at the same level like the images folder, so far so good, but it doesn't compile the spritemap into the imagesgen-folder. instead it puts it into "/images/foo/bar/", like it was before without the generated_images_dir configuration.

I'm not sure if it's a bug, i totally failed in setting the right configuration or compass just can't do it.

Does anyone has an idea?

**Comment by scottdavis** [comment]
what compass version i believe this has been fixed for next release/

**Comment by dirkschmid** [comment]
my Version is Compass 0.12.2

**Comment by scottdavis** [comment]
Im 99% sure this is fixed for next release 0.13 will be out soon
